karma.conf.js

```diff
@@ -0,0 +1,87 @@
+// Karma configuration
+// Generated on Fri May 06 2016 17:58:51 GMT+0530 (IST)
+
+module.exports = function (config) {
```

Comment:
@juslee , please help me in these kind of issues. They don't seem to be relevant .